src/CSF.SRDashboard.Client/Pages/CreateNewRequest.razor.cs

```diff
@@ -0,0 +1,104 @@
+﻿using CSF.SRDashboard.Client.DTO;
+using CSF.SRDashboard.Client.Services;
+using Microsoft.AspNetCore.Components;
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+using DSD.MSS.Blazor.Components.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CSF.SRDashboard.Client.Models;
+using CSF.SRDashboard.Client.DTO.WorkLoadManagement;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using CSF.SRDashboard.Client.PageValidators;
+
+namespace CSF.SRDashboard.Client.Pages
+{
+    public partial class CreateNewRequest
+    {
+        protected EditContext EditContext;
+
+        [Parameter]
+        public string Cdn { get; set; }
+
+        public string Comment { get; set; }
+
+        [Inject]
+        public IGatewayService GatewayService { get; set; }
+
+        [Inject]
+        public IWorkLoadManagementService WorkLoadService { get; set; }
+
+        [Inject]
+        public NavigationManager NavigationManager { get; set; }
```

Comment:
This is better so we can have all parameters and inject properties and then all public properties.
```suggestion
        [Inject]
        public IGatewayService GatewayService { get; set; }

        [Inject]
        public IWorkLoadManagementService WorkLoadService { get; set; }

        [Inject]
        public NavigationManager NavigationManager { get; set; }
        
         public string Comment { get; set; }
```

Reply:
Re-organized